worker/trash/trash.go

```diff
@@ -0,0 +1,32 @@
+package trash
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/cozy/cozy-stack/model/job"
+	"github.com/cozy/cozy-stack/model/vfs"
+)
+
+func init() {
+	job.AddWorker(&job.WorkerConfig{
+		WorkerType:   "trash-files",
+		Concurrency:  runtime.NumCPU(),
```

Comment:
This worker is reponsible for deleting files in swift. It means that it mainly waits swift's response to HTTP requests and doesn't consume much CPU nor RAM, so concurrency could be increased up to 3 or 5 * `NumCPU()` I think